### ISSUE TITLE
can't generate code error

### ISSUE THREAD

**Issue by WethersPulseMedia** [abi/screenshot-to-code#383]
finally got everything installed and my openai and anthropic keys added but get an error when I upload a 30 sec video. localhost shows everything is fine as well...
![image](https://github.com/user-attachments/assets/fb91bdeb-a960-4b7a-a47a-eb5a8122ff6a)
![image](https://github.com/user-attachments/assets/71363a4e-568d-4e57-a2cb-f551a70d9d22)

**Comment by WethersPulseMedia** [comment]
this is the error log 
![image](https://github.com/user-attachments/assets/634f5f38-b7d4-4984-acc7-04e59cdc9bc6)

**Comment by WethersPulseMedia** [comment]
also just noting i can clone pictures. just not video...

**Comment by abi** [comment]
Could you try with a shorter video?

**Comment by WethersPulseMedia** [comment]
i tried 10 seconds and same thing. tried to compress file too

**Comment by WethersPulseMedia** [comment]
it says permission denied any idea? 
![image](https://github.com/user-attachments/assets/488310d7-81db-4159-9cd5-fb522466d3e0)

**Comment by abi** [comment]
Yes, I think this is related to https://github.com/abi/screenshot-to-code/issues/311 It's a bug we need to fix. If you're a coder, you can try putting the processed files in a directory other than what TempFile puts it in and that should solve the problem.

I don't have an immediate fix for this issue.
